gufe/components/proteincomponent.py

```diff
@@ -56,7 +56,8 @@
 
 
 class ProteinComponent(ExplicitMoleculeComponent):
-    """Wrapper around a Protein representation.
+    """
+    :class:`Component` representing a protein.
```

Comment:
We really should rename this component - in reality this is just "a wrapper around something you placed in a PDB".

Not sure if we at least want to fix this in the docs?

Reply:
Fixed!

Reply:
c.f. https://github.com/OpenFreeEnergy/gufe/issues/139